battstat/battstat_applet.c

```diff
@@ -759,221 +775,224 @@ void
-    NULL
-   };
-
-  const gchar *documenters[] = {
-    "J\xC3\xB6rgen Pehrson <jp@spektr.eu.org>",
-    "Trevor Curtis <tcurtis@somaradio.ca>",
-    "Davyd Madeley <davyd@madeley.id.au>",
-    N_("MATE Documentation Team"),
-    NULL
-  };
-
-  char *comments = g_strdup_printf ("%s\n\n%s",
-		  _("This utility shows the status of your laptop battery."),
-		  power_management_using_upower () ?
-		    /* true */ _("upower backend enabled.") :
-			/* false */ _("Legacy backend enabled."));
+    const gchar *authors[] = {
+        "J\xC3\xB6rgen Pehrson <jp@spektr.eu.org>",
+        "Lennart Poettering <lennart@poettering.de> (Linux ACPI support)",
+        "Seth Nickell <snickell@stanford.edu> (GNOME2 port)",
+        "Davyd Madeley <davyd@madeley.id.au>",
+        "Ryan Lortie <desrt@desrt.ca>",
+        "Joe Marcus Clarke <marcus@FreeBSD.org> (FreeBSD ACPI support)",
+        NULL
+    };
+
+    const gchar *documenters[] = {
+        "J\xC3\xB6rgen Pehrson <jp@spektr.eu.org>",
+        "Trevor Curtis <tcurtis@somaradio.ca>",
+        "Davyd Madeley <davyd@madeley.id.au>",
+        N_("MATE Documentation Team"),
+        NULL
+    };
+
+    char *comments = g_strdup_printf ("%s\n\n%s",
+                                      _("This utility shows the status of your laptop battery."),
+                                      power_management_using_upower () ?
+                                      /* true */ _("upower backend enabled.") :
+                                      /* false */ _("Legacy backend enabled."));
 
 #ifdef ENABLE_NLS
-  const char **p;
-  for (p = documenters; *p; ++p)
-    *p = _(*p);
+    const char **p;
+    for (p = documenters; *p; ++p)
+        *p = _(*p);
 #endif
 
-  gtk_show_about_dialog( NULL,
-    "title",               _("About Battery Charge Monitor"),
-    "version",             VERSION,
-    "copyright",           _("Copyright \xc2\xa9 2000 The Gnulix Society\n"
-                             "Copyright \xc2\xa9 2002-2005 Free Software Foundation and others\n"
-                             "Copyright \xc2\xa9 2012-2020 MATE developers"),
-    "comments",            comments,
-    "authors",             authors,
-    "documenters",         documenters,
-    "translator-credits",  _("translator-credits"),
-    "logo-icon-name",      "battery",
-    NULL );
-
-  g_free (comments);
+    gtk_show_about_dialog (NULL,
+        "title",              _("About Battery Charge Monitor"),
+        "version",            VERSION,
+        "copyright",          _("Copyright \xc2\xa9 2000 The Gnulix Society\n"
+                              "Copyright \xc2\xa9 2002-2005 Free Software Foundation and others\n"
+                              "Copyright \xc2\xa9 2012-2020 MATE developers"),
+        "comments",           comments,
+        "authors",            authors,
+        "documenters",        documenters,
+        "translator-credits", _("translator-credits"),
+        "logo-icon-name",     "battery",
+        NULL);
```

Comment:
The arguments of gtk_show_about_dialog are not aligned.

Reply:
How?

Reply:
```diff
--- a/battstat/battstat_applet.c
+++ b/battstat/battstat_applet.c
@@ -848,17 +848,17 @@ about_cb (GtkAction *action, ProgressData *battstat)
 #endif
 
     gtk_show_about_dialog (NULL,
-        "title",              _("About Battery Charge Monitor"),
-        "version",            VERSION,
-        "copyright",          _("Copyright \xc2\xa9 2000 The Gnulix Society\n"
-                              "Copyright \xc2\xa9 2002-2005 Free Software Foundation and others\n"
-                              "Copyright \xc2\xa9 2012-2020 MATE developers"),
-        "comments",           comments,
-        "authors",            authors,
-        "documenters",        documenters,
-        "translator-credits", _("translator-credits"),
-        "logo-icon-name",     "battery",
-        NULL);
+                           "title",              _("About Battery Charge Monitor"),
+                           "version",            VERSION,
+                           "copyright",          _("Copyright \xc2\xa9 2000 The Gnulix Society\n"
+                                                   "Copyright \xc2\xa9 2002-2005 Free Software Foundation and others\n"
+                                                   "Copyright \xc2\xa9 2012-2020 MATE developers"),
+                           "comments",           comments,
+                           "authors",            authors,
+                           "documenters",        documenters,
+                           "translator-credits", _("translator-credits"),
+                           "logo-icon-name",     "battery",
+                           NULL);
 
     g_free (comments);
 }
```